app/sessionStore.js

```diff
@@ -658,7 +658,7 @@ module.exports.defaultAppState = () => {
         pinnedTopSites: pinnedTopSites
       },
       welcome: {
-        showOnLoad: true
+        showOnLoad: !['test', 'development'].includes(process.env.NODE_ENV)
```

Comment:
This is perfect! good call setting here (rather than putting the check into the code) 😄